lib/serviceBuilder.js

```diff
@@ -164,7 +164,12 @@ function makeCall(hostname, path, method, querystring, body, options = {}) {
 }
 
 function getHeader(options, miaHeaders, baseOptions) {
-  return Object.assign({}, miaHeaders || {}, baseOptions.headers || {}, options.headers || {})
+  const isMiaHeaderInjected = options.isMiaHeaderInjected !== undefined ? options.isMiaHeaderInjected : true
```

Comment:
why do you need this line?

Reply:
Because I have to manage the default case:` isMiaHeaderInjected === true` when `options.isMiaHeaderInjected` is missing

Reply:
`const isMiaHeaderInjected = options.isMiaHeaderInjected === undefined ? true : options.isMiaHeaderInjected` is more readable